### ISSUE TITLE
Ability to clone multiple files at once

### ISSUE THREAD

**Issue by VivaLosDoyers** [microsoft/AzureStorageExplorer#4555]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Problem

Our application locks output files, and often we need to download multiple files. We do this by cloning each file, downloading it, and then removing it. The clone option is only available when a single file is selected. It would really nice to be able to select multiple files and clone them.

### Desired Solution

Expose clone option when multiple files are select, and allow the user to choose a prefix or postfix to add to the cloned filenames.

**Comment by MRayermannMSFT** [comment]
@VivaLosDoyers why do you need to clone a leased (if that is not what you mean by locked, let us know) file in order to download it?

**Comment by VivaLosDoyers** [comment]
> @VivaLosDoyers why do you need to clone a leased (if that is not what you mean by locked, let us know) file in order to download it?

Thanks for asking this, it was a point I forgot to mention in the original request. Unfortunately the application in question doesn’t support deferred logging, and leasing the file first causes issues, so we can’t lease the file first.

**Comment by MRayermannMSFT** [comment]
@VivaLosDoyers unfortunately it is not feasible for us to implement multi-clone. An alternative solution you could do is:
1. Select the multilpe leased files you want to download
2. Use "Copy"
3. Create a new folder (like `copies-for-download-2024-05-20`)
4. Paste the files in there, effectively cloning them
5. Once the paste is done, download the clones
